pkg/components/components.go

```diff
@@ -463,7 +463,7 @@ func GetRole(namespace string) *rbacv1.Role {
 	return role
 }
 
-func GetClusterRole() *rbacv1.ClusterRole {
+func GetClusterRole(allowMultus bool) *rbacv1.ClusterRole {
```

Comment:
Thanks

nit: what about a new function that adds the rbac according needs, instead passing this argument ?

Reply:
I did it on a lower level.

Now when I read your comment again, I think you wanted the if to me moved to the templator instead?

Reply:
You did exactly as i meant :) thanks
unless you think there is a better way (i.e in the templator as you suggested)
this is good imho